music21/metadata/__init__.py

```diff
@@ -39,13 +39,99 @@
 
 .. image:: images/moduleMetadata-01.*
     :width: 600
+
+    A guide to this initial new implementation:
```

Comment:
Can this be phrased as "A guide to the 2022 Dublin Core implementation"  -- new will be relative.  I keep finding "new" from 2007 in our system and want to avoid adding it in.

Reply:
Done.